scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -2820,6 +2820,11 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) {
   }
 
   /* In addition to above named tuple patterns are allowed*/
+  object seqOKWithNamed extends SeqContextSensitive {
+    val isSequenceOK = true
+    override def isNamedTupleOk: Boolean = true
```

Comment:
why not add this to `seqOK` instead? if not possible, please add a test which demonstrates it.